developer/customization/articles/03-architecture/01-architecture-intro.markdown

```diff
@@ -0,0 +1,149 @@
+---
+header-id: architecture
+---
+
+# Architecture
+
+[TOC levels=1-4]
+
+@product@'s architecture helps you build sites that offer powerful functionality
+and terrific styling, by providing these benefits and more:
+
```

Comment:
Architecture is architecture; it doesn't help you build things. You would want to start with how well-designed Liferay's architecture is, what the parts are, and how they fit together seamlessly to provide a whole that is greater than the sum of its parts.